h/static/styles/mixins/_styled-text.scss

```diff
@@ -84,7 +84,7 @@
   blockquote {
     @include font-normal;
 
-    border-left: 3px solid $gray-lighter;
+    border-left: 3px solid $grey-3;
```

Comment:
Since we don't have $gray-lighter in the palette any more, I have replaced it with $grey-3.